build.gradle

```diff
@@ -60,7 +60,7 @@ allprojects {
     // we allow access to snapshots repo if ALLOW_SNAPSHOT_REPOSITORY is set, what means we are running on CI 
     // with Navigation Native forced to be some snapshot version
     // if you need to use snapshots while development, just set `addSnapshotsRepo` to true manually
-    def addSnapshotsRepo = System.getenv("ALLOW_SNAPSHOT_REPOSITORY")?.toBoolean() ?: false
+    def addSnapshotsRepo = true
```

Comment:
this intended?

Reply:
guess it'll be changed before merging :)

Reply:
That's right, when we upgrade to the new Maps SDK.